packages/bridge-status-controller/CHANGELOG.md

```diff
@@ -10,13 +10,23 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - **BREAKING:** Add `@metamask/snaps-controllers` peer dependency at `^9.19.0` ([#5634](https://github.com/MetaMask/core/pull/5634))
+- **BREAKING:** Add `@metamask/gas-fee-controller` peer dependency at `^23.0.0` ([#5643](https://github.com/MetaMask/core/pull/5643))
+- **BREAKING:** Add `@metamask/user-operation-controller` peer dependency at `^33.0.0` ([#5643](https://github.com/MetaMask/core/pull/5643))
+- **BREAKING:** Add `@metamask/assets-controllers` peer dependency at `^57.0.0` ([#5643](https://github.com/MetaMask/core/pull/5643))
+- Bump `@metamask/bridge-controller` peer dependency to `^15.0.0` ([#5643](https://github.com/MetaMask/core/pull/5643))
```

Comment:
This line can be removed from here as will be captured in the other change in line 26
```suggestion